api/src/main/java/com/okta/sdk/oauth2/OAuth2AccessToken.java

```diff
@@ -0,0 +1,56 @@
+package com.okta.sdk.oauth2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OAuth2AccessToken {
```

Comment:
This is an implementation-specific detail and should be moved to the `impl` module